tests/test_view.py

```diff
@@ -59,4 +60,156 @@ def test_pycon(self):
         self.assertFalse(view.mgr.pycon.python_redirect)
 
 
+def angle_axis(angle_deg, axis):
```

Comment:
Make a base class for the test case classes `ViewFPSCameraTC` and `ViewOrbitCameraTC` for holding `angle_axis()`.  An example can be found in https://github.com/solvcon/modmesh/blob/975f7c5730247f8e4ce770634ecb692fc3a70c0e/tests/test_universe.py#L35